### ISSUE TITLE
Feature Request: fine tuning 

### ISSUE THREAD

**Issue by michelgirault** [LlamaEdge/LlamaEdge#232]
### Summary

Would that be possible to have document on loading fine tuned model? is this possible ?

**Comment by alabulei1** [comment]
Yes, LlamaEdge can work with most models. If you use the same architecture from the base model, you can run your fine-tuned model directly. See this article for instructions. https://www.secondstate.io/articles/selfhost-huggingface-llms/

If the prompt template changes, you can add your prompt by re-building the repo. https://github.com/LlamaEdge/LlamaEdge/tree/main/crates/chat-prompts

**Comment by michelgirault** [comment]
thanks
